content/participants/ttcj@sarulathadurai/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Sarulatha Annadurai
+type: "participant"
+name: "Sarulatha Annadurai"
+portfolio: "(empty)"
+github-link: "https://github.com/sarulathadurai"
+linkedin-link: "https://www.linkedin.com/in/sarulatha-annadurai-060226180/"
+looking-for: "internship"
+description: "A complete beginner willing to put in whatever it takes!"
+---
+
+Hi! I am pursuing 3rd year of computer science engineering and would like to take up this challenge for expanding my knowledge and career.
+
+### introduction
+
+I am studious but I need guidance. With proper guidance I can do anything as there'd never be any hesitation of any sort from my side in terms of putting in the work.
+
+## why are you taking this challenge?
+
+I am willing to work as an intern to observe the work environment and learn how to work in a firm.
+I want to be strong in my domain.
+
+## something interesting about you?
+
+I can draw well.
+
+### help I need?
+
+I would need someone to give me correct ideas when I'm in trouble with my project.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/sarulat80587902)
+- [LinkedIn] (https://www.linkedin.com/in/sarulatha-annadurai-060226180/)
+- [gmail] (sarulathaannadurai@gmail.com)
```

Comment:
Use correct markdown format and also I would suggest adding **mailto:** before your email